app/actions/sessions.js

```diff
@@ -151,6 +154,67 @@ export function getScheduledQueries() {
     };
 }
 
+export function createScheduledQuery(connectionId, payload = {}) {
+  return (dispatch) => {
+    return dispatch(apiThunk(
+      'queries',
+      'POST',
+      'createScheduledQueryRequest',
+      payload.filename,
+      {
+        requestor: payload.requestor,
+        uids: payload.uids,
+        fid: payload.fid,
+        filename: payload.filename,
+        refreshInterval: payload.refreshInterval,
+        query: payload.query,
+        connectionId
+      }
+    )).then((res) => {
+      dispatch(createScheduledQueryAction(res));
```

Comment:
Is `createScheduledQueryAction` necessary? It doesn't seem to be used anywhere else.

If not, let's just do:
```js
dispatch({
    type: 'CREATE_SCHEDULED_QUERY',
    payload: res
});
```